lib/Runtime/Base/ScriptContext.h

```diff
@@ -52,6 +52,8 @@ class SRCINFO
     Js::ModuleID moduleID;
     ULONG grfsi;
 
+    Js::Var serializedScriptBuffer; // in order to keep a reference to serialized Script Buffer
```

Comment:
Looks SRCINFO has more impact than JsrtSourceHolder? SRCINFO also used for other inputs (Edge). JsrtSourceHolder only used for JsParseRunSerialized.